csrc/executor_utils.h

```diff
@@ -27,6 +27,39 @@
 #include <vector>
 
 namespace nvfuser {
+
+#define NVRTC_SAFE_CALL(x)                               \
+  do {                                                   \
+    nvrtcResult _result = x;                             \
+    if (_result != NVRTC_SUCCESS) {                      \
+      std::cerr << "\nerror: " #x " failed with error "  \
+                << nvrtcGetErrorString(_result) << '\n'; \
+      exit(1);                                           \
+    }                                                    \
+  } while (0)
+
+#define CUDA_SAFE_CALL(x)                                               \
+  do {                                                                  \
+    CUresult _result = x;                                               \
+    if (_result != CUDA_SUCCESS) {                                      \
+      const char* msg;                                                  \
+      cuGetErrorName(_result, &msg);                                    \
```

Comment:
Did you intend to use `cuGetErrorString`? Maybe use both `cuGetErrorName` and `cuGetErrorString` as in the CUDA RT version?

Reply:
Good point, will add that.